Library/Homebrew/tap.rb

```diff
@@ -399,6 +400,7 @@ def install(quiet: false, clone_target: nil, force_auto_update: nil,
 
     Commands.rebuild_commands_completion_list
     link_completions_and_manpages
+    Tap.tap_registry << self
```

Comment:
Double tap seems weird.

```suggestion
    Tap.registry << self
```